src/ablog/blog.py

```diff
@@ -293,15 +293,25 @@ def page_url(self, pagename):
             return url
 
 
-def html_builder_write_doc(self, docname, doctree):
+def html_builder_write_doc(self, docname, doctree, img_url=False):
     """
     Part of :meth:`sphinx.builders.html.StandaloneHTMLBuilder.write_doc` method
     used to convert *doctree* to HTML.
+
+    Extra argument `img_url` enables conversion of `<img>` source paths to
+    fully qualified URLs based on `blog_baseurl`.
     """
+    # source of images
+    img_folder = "_images"
```

Comment:
Is this always going to be true?

Reply:
yes, it is also hardcoded to this name in sphinx:

https://github.com/sphinx-doc/sphinx/blob/2c83af0aab7080e0b78d4a16981eed878b2cac4c/sphinx/builders/html/__init__.py#L663